order-routing/javascript/rankers/default/src/index.test.liquid

```diff
@@ -0,0 +1,68 @@
+{%- if flavor contains "vanilla-js" -%}
+import { describe, it, expect } from 'vitest';
+import orderRoutingLocationRule from './index';
```

Comment:
been a while since I used JS - but is this just giving a name to the otherwise anonymous function exported by default by `index.js/ts`?

Reply:
yes, it gives the name to the exported anonymous function.